go/vt/sqlparser/ast.go

```diff
@@ -724,6 +724,7 @@ func (node *Set) walkSubtree(visit Visit) error {
 type DBDDL struct {
 	Action   string
 	DBName   string
+	Trigger  string
```

Comment:
Rather have this on DDL, similar to View. Just add a Trigger there, or reuse TriggerSpec with just the name filled in